resources/views/includes/status.blade.php

```diff
@@ -1,20 +1,13 @@
-<div class="card status mt-3" id="status-{{ $status->id }}" data-body="{{ $status->body }}">
-
-    @if (empty($currentUser))
-        <?php $currentUser = Auth::user(); ?>
-    @endif
-
+<h5 class="mt-4 dateHeading"></h5>
+<div class="card status mt-3" id="status-{{ $status->id }}" data-body="{{ $status->body }}" data-date="{{formatNewDay(new \DateTime($status->trainCheckin->departure)) }}">
     @if (Route::current()->uri == "status/{id}")
-        <?php $mapLines = $status->trainCheckin->getMapLines(); ?>
-        @if($mapLines != "[]")
+        @if($status->trainCheckin->HafasTrip->polyline)
         <div class="card-img-top">
-            <div id="map-{{ $status->id }}" class="map statusMap embed-responsive embed-responsive-16by9" data-polygon="{{ $mapLines }}"></div>
+            <div id="map-{{ $status->id }}" class="map statusMap embed-responsive embed-responsive-16by9" data-polygon="{{ $mapLines = $status->trainCheckin->getMapLines() }}"></div>
```

Comment:
Var-Zuweisung wird dann nicht mehr benötigt.

```suggestion
            <div id="map-{{ $status->id }}" class="map statusMap embed-responsive embed-responsive-16by9" data-polygon="{{ $status->trainCheckin->getMapLines() }}"></div>
```